protocol-units/settlement/mcr/contracts/src/token/faucet/MOVEFaucet.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
```

Comment:
You got a link to what this was based on?

Reply:
https://github.com/marvinkruse/simple-faucet/blob/master/smartcontract/faucet.sol